trucks.js

```diff
@@ -510,19 +510,13 @@ function getTrucks() {
 }
 
```

Comment:
Your trucks module is missing methods for getFoodTypes() and filterByFoodType(type)